peps/pep-0242.rst

```diff
@@ -3,11 +3,20 @@ Title: Numeric Kinds
 Author: Paul F. Dubois <paul@pfdubois.com>
 Status: Rejected
 Type: Standards Track
-Content-Type: text/x-rst
 Created: 17-Mar-2001
 Python-Version: 2.2
 Post-History: 17-Apr-2001
 
+.. rejected::
```

Comment:
If the "PEP has been closed by the author", rather than pronounced upon, then shouldn't this be "withdrawn" as well?

```suggestion
.. withdrawn::
```